app/forms/charityInformation/CharityContactDetailsFormProvider.scala

```diff
@@ -33,8 +33,7 @@ class CharityContactDetailsFormProvider @Inject() extends Mappings {
       mapping(
         "mainPhoneNumber" -> text("charityContactDetails.mainPhoneNumber.error.required")
           .verifying(regexp(validateTelephoneNumber,"charityContactDetails.mainPhoneNumber.error.format")),
-        "alternativePhoneNumber" -> optional(text()
-          .verifying(regexp(validateTelephoneNumber,"charityContactDetails.alternativePhoneNumber.error.format"))),
+        "alternativePhoneNumber" -> optional(text()),
```

Comment:
you appear to have removed  the formatting error for the alternative phone number?